stackcollapse-phpspy.pl

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/perl -w
```

Comment:
The `use warnings;` line covers the -w usage, this can be removed. Would maybe sub in `/usr/bin/env perl` to play nicer with custom perl installs